likes/views.py

```diff
@@ -1,3 +1,65 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http          import JsonResponse
+from django.views         import View
+from json.decoder         import JSONDecodeError
+
+from products.models      import Product
+from likes.models         import Like
+from handwash.utils       import login_decorator
+
+class LikeView(View):
+  @login_decorator
+  def post(self, request):
+    try:
+      data = json.loads(request.body)
+      user = request.user
+
+      product_id = data['product_id']
+
+      if not product_id:
+        return JsonResponse({'MESSAGE' : 'KEY_ERROR'}, status = 400)
+
+      if not Product.objects.filter(id=product_id).exists():
+        return JsonResponse({'MESSAGE' : 'PRODUCT_DOES_NOT_EXIST'}, status = 404)
+
+      product = Product.objects.get(id=product_id)
```

Comment:
지금 코드도 정말 잘 작성해 주셨습니다! 당장 merge해도 될 것 같습니다!

그대로 피드백을 드리자면 아래와 같이 코드를 수정하면 어떨까요?
`get_or_create()` method를 사용하면 product 존재 여부를 판단해야되는 아래 코드도 필요 없이 line 18~35 줄을 5줄 정도로 바꿀 수 있을 것 같아요!

```suggestion
      like, created = Like.objects.get_or_create(user = user, product_id = data['product_id'])
      
      if not created:
          like.delete()
      
      return JsonResponse({'MESSAGE' : 'SUCCESS'}, status = 200)

    except KeyError:
        return JsonResponse({'MESSAGE' : 'KEY_ERROR'}, status = 400)    
```